metadata/service/server/init.yml

```diff
@@ -0,0 +1,24 @@
+applications:
+- rsyslog
+classes:
+- service.rsyslog.support
+parameters:
+  _param:
+    rsyslog_port: 10514
+  rsyslog:
+    server:
+      enabled: true
+      module:
+        imtcp: {}
+      template:
+        RemoteStorage:
+          parameter:
+            type: string
+            string: /var/log/%HOSTNAME%/%programname%.log
+      ruleset:
+        RemoteLogs:
+          description: action(type="omfile" dynaFile="RemoteStorage")
```

Comment:
"action" format is actually standart, but ruleset does not. An example would be
```
ruleset(name="remote10516"){
    if prifilt("mail.*") then {
        /var/log/mail10516
        stop
    }
    /var/log/remote10516
}
```
That is why i've made decision to create field  "description"  and use instead